### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by gabztcr**
- Update all fields from the 3 component
- Include list, find and clear commands for contacts
- Edit help window

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -69,123 +75,145 @@ Format: `help`
 Switches to either `contact`, `meeting` or `deliverable` mode.
 
 Format: `switch MODE`
-* Subsequent commands will be with respect to managing the mode.
+* Subsequent commands will be with respect to the `MODE`.
```
Comment (MerlinLim) [inline docs/UserGuide.md:78]: Also note down what happens when user switches mode (e.g list of items are displayed)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -15,27 +18,29 @@ can get your product management tasks done faster than traditional GUI apps.
...
+Note that the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`switch contact`** and pressing Enter will switch to the Contact mode.<br>
-   Some example commands you can try:
+1. Type a command in the command box and press Enter to execute it. 
+Here are some example commands you can try:
 
-   * **`switch deliverable`** : Switches to `deliverable` mode.
+   * **`switch`**`deliverable` : Switches to `deliverable` mode.
```
Comment (MerlinLim) [inline docs/UserGuide.md:35]: Consider adding whitespace between words (e.g **`switch`** `deliverable`...)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -69,123 +75,145 @@ Format: `help`
...
 Adds a deliverable to the deliverable list.
 
-Format: `add t/TITLE [desc/DESCRIPTION] [dead/DEADLINE] [c/CONTACT]…`​`
+Format: `add t/TITLE m/MILESTONE [d/DESCRIPTION] [by/DEADLINE] [c/CONTACTS]`
 * `TITLE` is the main heading of the deliverable.
-* `DESCRIPTION` is the additional details of the deliverable (e.g. sub-requirements). 
+* `MILESTONE` is the milestone tagged to the deliverable.
+* `MILESTONE` takes in numerical values separated by periods, e.g. `1.3`, `14.2.1`.
+* `DESCRIPTION` contains additional information about the deliverable, e.g. sub-requirements. 
 * `DEADLINE` is the due date time of the deliverable in DD-MM-YYYY HH:MM format.
```
Comment (MerlinLim) [inline docs/UserGuide.md:169]: Consider using lower caps 'm' to differentiate between minutes and months (e.g `DD-MM-YYYY HH:mm`)

File: docs/UserGuide.md [hunk 4]
```diff
@@ -194,53 +222,53 @@ Examples:
 
 Adds a meeting to the meeting list.
 
-Format: `add t/TITLE [desc/DESCRIPTION] from/FROM [to/TO] [c/CONTACTS] [l/LOCATION]`
+Format: `add t/TITLE [d/DESCRIPTION] from/FROM to/TO [c/CONTACTS] [l/LOCATION]`
 * `TITLE` is the main heading of the meeting.
-* `DESCRIPTION` is the additional details of the meeting (e.g. agenda). 
+* `DESCRIPTION` contains additional details about the meeting, e.g. agenda. 
 * `FROM` is the start date time of the meeting in DD-MM-YYYY HH:MM format.
 * `TO` is the end date time of the meeting in DD-MM-YYYY HH:MM format.
```
Comment (MerlinLim) [inline docs/UserGuide.md:229]: Sample consideration as in deliverables

File: docs/UserGuide.md [hunk 5]
```diff
@@ -15,27 +18,29 @@ can get your product management tasks done faster than traditional GUI apps.
...
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`switch contact`** and pressing Enter will switch to the Contact mode.<br>
-   Some example commands you can try:
+1. Type a command in the command box and press Enter to execute it. 
+Here are some example commands you can try:
 
-   * **`switch deliverable`** : Switches to `deliverable` mode.
+   * **`switch`**`deliverable` : Switches to `deliverable` mode.
 
-   * **`add`**`stk n/Betty Clarke p/98765432` : Adds a contact named `Betty Clarke` to Productiv.
+   * **`add`**`r/stk n/Betty Clarke p/98765432` : If in `contact` mode, adds a contact named `Betty Clarke`.
```
Comment (chrystalquek) [inline docs/UserGuide.md:37]: email is a compulsory field (phone is an optional field)

File: docs/UserGuide.md [hunk 6]
```diff
@@ -3,10 +3,13 @@ layout: page
...
+*Productiv* is a desktop app that assists product managers like yourself with organising product-related information 
+(**contacts**, **deliverables** and **meetings**) so that you can track your product's development more comprehensively 
+and conveniently.
+
+*Productiv* is optimized for use via Command Line Interface (CLI), but still has the benefits of a Graphical User 
+Interface (GUI). Thus, if you can type fast, *Productiv* can get your product management tasks done faster than 
+traditional GUI apps.
 
 * Table of Contents
 {:toc}
```
Comment (MerlinLim) [inline docs/UserGuide.md:15]: Consider removing this section if there's no intention of updating Table of Contents for now.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -250,25 +278,30 @@ Format: `exit`
...
 
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
-**Mode** | `mode`
-**Add (Contact)** | `add ROLE n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> e.g., `add stk n/Jenny Ho p/98981212 e/jennyho@pmail.com`
-**Edit (Meeting)** | `edit INDEX [t/TITLE] [desc/DESCRIPTION] [from/FROM] [to/TO] [c/CONTACTS] [l/LOCATION]` <br> e.g., `edit 2 l/Blk C Level 8 Meeting Room 1`
-**Mark Deliverable as Done** | `done INDEX` <br> e.g., `done 2`
+**Switch** | `switch MODE` <br> e.g., `switch deliverable`
```
Comment (MerlinLim) [inline docs/UserGuide.md:298]: The commas in `e.g.,` seem to be a typo. I could be missing some markdown formatting.

**Review by MerlinLim — COMMENTED** [review]
The structure looks good and descriptions of each mode are correct. Just some minor details for your consideration.

File: docs/UserGuide.md (hunk 4 above)
Comment (MerlinLim) [reply docs/UserGuide.md:229]: Same* consideration

File: docs/UserGuide.md (hunk 2 above)
Comment (gabztcr) [reply docs/UserGuide.md:35]: I just tried and the whitespace doesn't get reflected on markdown. The current implementation should work alright (there's an illusion of whitespace). Thanks for the suggestion 😸 !

**Review by MerlinLim — APPROVED** [review]
LGTM!

**Pull request merged**